adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/commands/CancelTickets.java

```diff
@@ -0,0 +1,16 @@
+package com.microservicesteam.adele.ticketmaster.commands;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface CancelTickets extends TicketsCommand {
```

Comment:
Question: since `CancelTickets` command extends `TicketsCommand`, does this mean that whoever issues this command have to supply the positions as well, not enough to say which booking should be cancelled? In other words, is it possible to partially cancel a booking (e.g. just 1 seat out of the 3 previously booked ones)?